index.rst

```diff
@@ -54,24 +54,8 @@ throughout the communinity.
 What is EM GeoSci?
 ------------------
 
-The emphasis is on  physical principles of electromagnetic methods and their
-application in solving problems. Rigorous analytic solutions of the EM
-problems with specific transmitting fields and earth models is left to
-existing books and research papers. Also, the resource does not concentrate
-upon the scientific computing aspects to numerically solve Maxwell’s
-equations. We do however, make much use of software that can perform those
-functions. Rather, our emphasis is on using analytic and numerical solutions
-to understand electromagnetic  fields and fluxes obtained from various types
-of transmitters in different geological environments.
-
-Our emphasis is on fundamentals and applications and correspondingly
-:ref:`case_histories_index` play an essential role. These provide the
-motivation for using electromagnetics and show the success, or not, of their
-application in making an impact upon the problem at hand.
-
-Examples are available through `Jupyter Notebooks`_ and can be run through Binders_.
-The simulations use the open-source Simulation and Parameter Estimation package
-SimPEG_.
+EM GeoSci is an open-source resource that emphasizes the physical principles of electromagnetic methods and their application in solving geoscience problems. Rigorous analytic solutions of the EM problems with specific transmitting fields and earth models are left to existing books and research papers, and so too are the computational details required to numerically solve Maxwell’s equations. Our emphasis is on using analytic and numerical solutions to understand electromagnetic fields and fluxes obtained from various types of transmitters in different geological environments and to show how different EM surveys, using controlled or natural sources, can be applied in practise. `Case Histories <https://em.geosci.xyz/content/case_histories/index.html#case-histories-index>`_ play an essential role. They allow readers to make the connection between the geoscience problem, the EM survey, and the eventual outcome. Understanding the fundamental physics is facilitated through the use of `Jupyter Notebooks <https://mybinder.org/v2/gh/geoscixyz/em-apps/master?filepath=index.ipynb>`_ which allow users to visualize fields and explore understanding by changing parameters. The simulations use the open-source package `SimPEG <https://simpeg.xyz>`_ (Simulation and Parameter Estimation).
```

Comment:
For SimPEG, I will extend the parenthetical to be the whole name ("Simulation and Parameter Estimation in Geophysics")